styleguide/source/assets/scss/02-molecules/_jama.scss

```diff
@@ -26,17 +26,29 @@
 
   .ama__link--with-image {
     img {
-      display: none;
+      @include breakpoint($bp-xs max-width) {
+        width: 100%;
+      }
     }
 
     a {
+      @include breakpoint($bp-xs max-width) {
```

Comment:
I have this as the default then override it with something like 
`@include breakpoint($bp-small) {
  width: auto;
}`

Reply:
The rule of thumb is mobile values are default and gets overridden for bigger size viewports

Reply:
Thanks @designerneil - I have updated the pr with your recommendations. Let me know, if this approach works.